src/master/mac.rs

```diff
@@ -4,6 +4,9 @@ use crate::{ClipboardHandler, CallbackResult, Master};
 use objc::runtime::{Object, Class};
 use objc_id::{Id};
 
+#[link(name = "AppKit", kind = "framework")]
```

Comment:
interesting, `objc` doesn't link it?

Reply:
I'm not really an expert on this, but it wasn't linked until I used this on my mac (Big Sur 11.2). It is also used in many clipboard crates.

Reply:
Understood